website/pages/en/substreams/[[...slug]].mdx

```diff
@@ -22,20 +22,32 @@ export async function getStaticProps({ params: { slug = ['README'] } }) {
   if (response.status === 404 && paths !== 'README') {
     response = await fetch(`${baseURL}${paths}/README.md`)
   }
-  const data = await response.text()
+  const data = (await response.text())
+    // replace {% embed ... %} with <iframe />
+    .replaceAll(
+      /{%\s+embed\s+url="(.*?)"\s+%}/g,
+      (...m) =>
+        `<iframe src="${m[1].replace(
+          // we need enhance YouTube links, otherwise they will be not loaded in iframe
+          'youtube.com/watch?v=',
+          'youtube.com/embed/'
+        )}" style={{aspectRatio: 16/9, width: '100%'}}/>`
+    )
+    // remove gitbook {% ... %} elements
+    .replaceAll(/{%.*?%}/g, '')
+    // close img tags only if he doesn't point to .gitbook
+    .replaceAll(/<img(.*?)>/g, (...m) => (m[1].includes('src=".gitbook') ? '' : `<img${m[1]}/>`))
+    // fixes http://localhost:3000/en/substreams/reference-and-specs/authentication/
+    .replaceAll('<pre class="language-bash" data-overflow="wrap"><code class="lang-bash">', '```bash\n')
+    // fixes http://localhost:3000/en/substreams/developers-guide/modules/inputs/
+    .replaceAll('<pre class="language-yaml" data-title="manifest excerpt"><code class="lang-yaml">', '```yaml\n')
+    .replaceAll('</code></pre>', '```')
   const mdx = await buildDynamicMDX(data, {
     mdxOptions: {
-      format: 'md',
+      format: paths.endsWith('/change-log') ? 'md' : 'mdx',
```

Comment:
Why this?

Reply:
due to this line https://github.com/streamingfast/substreams/blob/f7bfe8fc45f860a935c2346bcd43b7b35bd76430/docs/release-notes/change-log.md?plain=1#L139 mdx2 treat `{Type}` as injection of variable, I guess we should use md parser everywhere for gitbooksdocs